RecoHI/HiEgammaAlgos/python/RecoHiEgamma_EventContent_cff.py

```diff
@@ -1,104 +1,79 @@
-    'keep floatedmValueMap_hiDetachedTripletStepSelector_MVAVals_*',    
-    'keep floatedmValueMap_hiGeneralTracks_MVAVals_*',                  
-    'keep floatedmValueMap_hiInitialStepSelector_MVAVals_*',            
-    'keep floatedmValueMap_hiLowPtTripletStepQual_MVAVals_*',           
-    'keep floatedmValueMap_hiLowPtTripletStepSelector_MVAVals_*', 
-    'keep floatedmValueMap_hiPixelPairStepSelector_MVAVals_*',          
-    'keep floatedmValueMap_hiRegitMuInitialStepSelector_MVAVals_*', 
-    'keep floatedmValueMap_hiRegitMuMixedTripletStepSelector_MVAVals_*',
-    'keep floatedmValueMap_hiRegitMuPixelLessStepSelector_MVAVals_*', 
-    'keep floatedmValueMap_hiRegitMuPixelPairStepSelector_MVAVals_*', 
-    #'keep recoPFCandidates_*_*_*',
-    'keep recoPFCandidates_particleFlowEGamma_*_*',
-    'keep recoPFCandidates_particleFlowTmp_*_*',
-    "drop recoPFClusters_*_*_*",
-    #"keep recoElectronSeeds_*_*_*",
-    "keep recoElectronSeeds_ecalDrivenElectronSeeds_*_*", 
-    #"keep recoGsfElectrons_*_*_*",
-    "keep recoGsfElectrons_ecalDrivenGsfElectrons_*_*",
-    "keep recoGsfElectrons_electronsWithPresel_*_*",
-    "keep recoGsfElectrons_gedGsfElectronsTmp_*_*",
-    "keep recoGsfElectrons_mvaElectrons_*_*",
-    'keep recoHIPhotonIsolationedmValueMap_photonIsolationHIProducer_*_*',
-    'keep recoPhotons_gedPhotonsTmp_*_*',
-    'keep recoHIPhotonIsolationedmValueMap_photonIsolationHIProducerGED_*_*',
-    'keep recoElectronSeeds_ecalDrivenElectronSeeds_*_*',
-     'keep recoTrackExtras_electronGsfTracks_*_*'
-    )
-    )
+        'keep recoSuperClusters_correctedHybridSuperClusters_*_*',        
+        'keep recoSuperClusters_correctedMulti5x5SuperClustersWithPreshower_*_*',
+        'keep recoSuperClusters_hfEMClusters_*_*',
+        'keep recoSuperClusters_hybridSuperClusters_*_*',
+        'keep recoSuperClusters_multi5x5SuperClusters_*_*',
+        'keep recoSuperClusters_multi5x5SuperClustersWithPreshower_*_*',
+        'keep recoSuperClusters_particleFlowEGamma_*_*',
+        'keep recoSuperClusters_particleFlowSuperClusterECAL_*_*',
+        'keep recoSuperClusters_particleFlowSuperClusterOOTECAL_*_*',
+        'keep recoSuperClusters_lowPtGsfElectronSuperClusters_*_*',
+        'keep recoCaloClusters_hfEMClusters_*_*',
+        'keep recoCaloClusters_particleFlowEGamma_*_*',
+        'keep recoCaloClusters_cleanedHybridSuperClusters_*_*',
+        'keep recoCaloClusters_hybridSuperClusters_*_*',
+        'keep recoCaloClusters_uncleanedHybridSuperClusters_*_*',
+        'keep recoCaloClusters_islandBasicClusters_*_*',
+        'keep recoCaloClusters_multi5x5BasicClustersCleaned_*_*',
+        'keep recoCaloClusters_multi5x5BasicClustersUncleaned_*_*',
+        'keep recoCaloClusters_multi5x5SuperClusters_*_*',
+        'keep recoCaloClusters_particleFlowSuperClusterECAL_*_*',
+        'keep recoCaloClusters_particleFlowSuperClusterOOTECAL_*_*',
```

Comment:
This and the following recoCaloClusters were not there in the original event content